stylesheets/scss/_layout.scss

```diff
@@ -2,3 +2,27 @@
 	Layout goes in this file. Susy setup should go here too.
 	Layout classes should be prefixed `.l-`
 */
+
+$susy: (
+  columns: $column-count,
+  use-custom: (rem:true),
+  global-box-sizing: border-box,
+  gutter-position: inside-static,
+  column-width: 96px,
+  gutters: $gutter-column/96px,
+  // debug: (
+  //   image: show-columns,
+  //   output: overlay,
+  //   toggle: bottom left,
+  // )
+);
+
+.l-container {
+  @include container(1200px);
```

Comment:
I don't think passing in a pixel value is part of `container()`, at least not documented that I see. I think the idea is the width is based on `column-width` and `gutters`. Want to go with just `@include container();`?